plugins/database/src/db_plugin.cpp

```diff
@@ -15300,6 +15347,10 @@ irods::database* plugin_factory(
         DATABASE_OP_PURGE_SERVER_LOAD_DIGEST,
         function<error(plugin_context&,const char*)>(
             db_purge_server_load_digest_op ) );
+    pg->add_operation(
+        DATABASE_OP_SET_DELAY_SERVER,
```

Comment:
Oooops, this got replaced by a different operation in a later commit. Please disregard.

Reply:
well, it's setting the mechanism in motion, not setting the leader directly.   so, one vote for leaving it as is.

however, i think this is now moot because of kory's more generic update mechanism ... in a later commit